test/nlp/slack-model.jl

```diff
@@ -7,84 +7,86 @@
     J(x) = [1.0 -2.0 0; -0.5x[1] -2.0x[2] -1]
     H(x, y) = H(x) + y[2] * [-0.5 0 0; 0 -2.0 0; 0 0 0]
 
-    nlp = SlackModel(SimpleNLPModel())
-    n = nlp.meta.nvar
-    m = nlp.meta.ncon
+    for dt in [Float64, Float32]
```

Comment:
You can plug this `for` on the `@testset` ([doc](https://docs.julialang.org/en/v1/stdlib/Test/#Test.@testset)) so you can avoid the indentation change again.
Also, you can set the type of `H(x)` and `J(x)`, although it doesn't seem to affect the tests.
```suggestion
    for T in [Float64, Float32]
```

Reply:
Thanks for the link, I never used that before.

There isn't any test on the types in this package, and I have a feeling not all the models would pass. We should make it a new issue for a future PR. What do you think?

Reply:
I don't think we need to test types for now. Probably with the solver revamp we'll find out what we need.